recipes/xkbcommon/all/conanfile.py

```diff
@@ -82,7 +82,7 @@ def build_requirements(self):
             self.tool_requires("pkgconf/2.0.3")
         if self._has_build_profile and self.options.get_safe("with_wayland"):
             self.tool_requires("wayland/<host_version>")
-            self.tool_requires("wayland-protocols/1.31")
+            self.tool_requires("wayland-protocols/1.32")
```

Comment:
```suggestion
            self.tool_requires("wayland-protocols/<host_version>")
```

https://docs.conan.io/2/reference/conanfile/methods/build_requirements.html#host-version